test/config/test_config.py

```diff
@@ -21,9 +15,17 @@ def loaded_config(tmp_path):
         "columns": { "vehicle_number": { "max": "NA", "min": 0 }, "maximum_speed" : {"max" : 150, "min" : 0}, "service_date" : {"max" : "NA", "min" : "1990-01-01"} }
     }
 
+
+@pytest.fixture
+def empty_config():
+    return config
```

Comment:
You should return a new instance of Config() instead. This config is the same config used in main code. This config is not guaranteed to be empty in the future. And because this is a reference to a single config, you might modify this config in some test and be using the modified config in other tests.

Reply:
Upon some testing, this is exactly the case. empty_config() and loaded_config both uses the same instance of config, meaning when you use the empty_config fixture, it returns a config with data already loaded in it, and I imagine you do not want that. And of course, the loaded_config fixture also returns a previously modified instance of config, and I'm sure you do not want that either.

You should create a new instance of Config() for both empty_config() and loaded_config() and return that instead of returning `config`.